recipes/pylhe/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pylhe" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pylhe-{{ version }}.tar.gz
+  sha256: 75d179d8abf21ea2686bb25d9e86e2f3c91cce5c251c3deb267ee6ac46c7e439
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
```

Comment:
Should also add `graphviz` dependency.

c.f. https://github.com/scikit-hep/pylhe/blob/c0d6c1b54039eac715fca5b18696fc82619c2da9/.github/workflows/ci.yml#L42

Reply:
Am going to be offline. Very quickly - for meta.yml you need `python-graphviz`, I believe, as I have in DecayLanguage.